test/NVQPP/mapping_test-1.cpp

```diff
@@ -6,9 +6,11 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: nvq++ %cpp_std -v %s -o %t --target oqc --emulate && CUDAQ_DUMP_JIT_IR=1 %t |& FileCheck %s
+// REQUIRES: c++20
+
+// RUN: nvq++ %s -o %t --target oqc --emulate && CUDAQ_DUMP_JIT_IR=1 %t | FileCheck %s
+// RUN: nvq++ %s -o %t --target iqm --iqm-machine Adonis --mapping-file "%p/../Supplemental/Adonis Variant.txt" --emulate && %t
 // RUN: nvq++ -std=c++17 --enable-mlir %s -o %t
```

Comment:
How does this required c++20 and yet still have a `-std=c++17` version of this test?

Reply:
There isn't a link to the failures in the issue.

This test will compile using the `-std=c++17` option on a correctly configured machine with a C++20 compiler. So that is what is being tested.

Whether the binary will execute on a malconfigured system is a different question.